dist/rpm/update-cache.sh

```diff
@@ -1,5 +1,8 @@
 #! /bin/sh
 
+# Note: this might not work correctly anymore because the 'lr' command
+# was obsoleted.
```

Comment:
Is there documented anywhere? I can't find any mention of it in https://github.com/search?q=repo%3AopenSUSE%2Fobs-docu+localrun

Reply:
If you call `osc service lr`, it will tell you.

Reply:
Like @baierjan wrote, `osc help service` tells me, but I don't know what I can use as a replacement.
I don't know what the update-cache.sh script is supposed to do. When I ran it, more files than expected were changed.

Also, in this case the script cannot be used because it would do `cd openQA` and generate the tar file from the existing asset urls from the version of openQA in OBS, but I needed the changed version from my local update.

Reply:
```
$ osc service lr
WARNING: Command 'lr' is obsolete, please use 'run' instead.